model/channel_member_test.go

```diff
@@ -32,7 +32,7 @@ func TestChannelMemberIsValid(t *testing.T) {
 	o.NotifyProps = GetDefaultChannelNotifyProps()
 	o.UserId = NewId()
 	if err := o.IsValid(); err == nil {
-		t.Fatal("should be invalid")
+		require.Fail(t, "should be invalid")
```

Comment:
This is going to generate a old pattern approach without been catch by the go vet checker if we remove the comment of that part, I would suggest to migrate the code properly to a semantic assertion (`require.Nil(t, err)`)